### PR TITLE
Photo backend and testcases

### PR DESCRIPTION
**Pull request by Nil-Cipher**
Added database connector to the Photo collection, relevant data classes, http endpoints, and test cases.

### CODE REVIEW

File: ZenJourney/backend/src/test/kotlin/com/backend/ApplicationTest.kt [hunk 1]
```diff
@@ -18,4 +30,62 @@ class ApplicationTest {
             assertEquals("Hello World!", bodyAsText())
         }
     }
+
+    @Test
+    fun testPhoto_PostFindDelete() = testApplication {
+        application {
+            configureRouting()
+        }
```
Comment (Carson-Tang) [inline ZenJourney/backend/src/test/kotlin/com/backend/ApplicationTest.kt:38]: maybe in a @BeforeTest or just for future/fyi

File: ZenJourney/backend/src/test/kotlin/com/backend/fire.png
Comment (Carson-Tang) [inline ZenJourney/backend/src/test/kotlin/com/backend/fire.png:1]: is this being used in a test

**Comment by Carson-Tang** [comment]
probably .gitignore the .idea files

**Review by Carson-Tang — APPROVED** [review]

**Comment by Carson-Tang** [comment]
rocket

**Comment by victoranyun** [comment]
> probably .gitignore the .idea files

+1

File: ZenJourney/backend/src/main/kotlin/com/backend/application/request/PhotoRequest.kt
Comment (alice41abc) [inline ZenJourney/backend/src/main/kotlin/com/backend/application/request/PhotoRequest.kt:1]: need to move this to shared folder

File: ZenJourney/backend/src/main/kotlin/com/backend/application/response/ListResponse.kt
Comment (alice41abc) [inline ZenJourney/backend/src/main/kotlin/com/backend/application/response/ListResponse.kt:1]: need to move this to the shared folder

File: ZenJourney/backend/src/main/kotlin/com/backend/application/response/PhotoResponse.kt
Comment (alice41abc) [inline ZenJourney/backend/src/main/kotlin/com/backend/application/response/PhotoResponse.kt:1]: need to move this to the shared folder

File: ZenJourney/shared/src/main/kotlin/user/Photo.kt
Comment (alice41abc) [inline ZenJourney/shared/src/main/kotlin/user/Photo.kt:1]: can you move this to its own ZenJourney/shared/src/main/kotlin/journal folder, along with the request and response files

**Review by alice41abc — CHANGES_REQUESTED** [review]

**Pull request merged**